### PR TITLE
Feature/always create null data vcenter

### PR DESCRIPTION
**Pull request by nicolearagao**
Creates template for Vcenter facts which contain all facts regardless if they are found or not. Their values are set to None and later updated when fact is found.

- Modifies results_template name and test_all_facts_with_null_value name
- Creates classes of RAW FACTS
- Creates raw_facts_template to replace empty facts list
- Use Alias name instead of raw_facts_template()
- Create Test to make sure fields are being filled with null if fact is not present

### CODE REVIEW

File: quipucords/fingerprinter/task.py [hunk 1]
```diff
@@ -1369,9 +1369,10 @@ def _process_vcenter_fact(self, source, fact):
         if vcenter_os_release != '':
             rhel_os_releases = ['red hat enterprise linux', 'rhel']
             for rhel_release in rhel_os_releases:
-                if rhel_release in vcenter_os_release.lower():
-                    is_redhat = True
-                    break
+                if vcenter_os_release is not None:
```
Comment (bruno-fs) [inline quipucords/fingerprinter/task.py:1372]: you can avoid changing this whole block by replacing the call on line 1367

```
vcenter_os_release = default_getter(fact, 'vm.os', '')
```

**Review by bruno-fs — COMMENTED** [review]
Awesome work! :clap: 

I'm suggesting 2 small changes and would like to know what you think about refactoring proposal on the constants

**Review by gtanzillo — APPROVED** [review]
Looks great, awesome work!

**Review by bruno-fs — APPROVED** [review]
LGTM :+1: :heavy_check_mark:

**Review by gtanzillo — APPROVED** [review]

**Review by bruno-fs — APPROVED** [review]

**Pull request merged**